modules/websearch/lib/search_engine.py

```diff
@@ -4158,6 +4163,10 @@ def rank_records(req, rank_method_code, rank_limit_relevance, hitset_global, pat
     else:
         related_to = pattern
 
+    if CFG_WEBSEARCH_OBELIX_REDIS:
+        if rank_method_code == "rrm":
```

Comment:
yes, but it's done like this to make it clear that this will only happen if CFG_WEBSEARCH_OBELIX_REDIS is set